code/game/gamemodes/nations/flagobjs.dm

```diff
@@ -0,0 +1,53 @@
+//Nations/Department flags --Nations Gamemode Specific
+/obj/item/flag/nation/cargo
+	name = "Cargonia flag"
+	desc = "The flag of the independent, sovereign nation of Cargonia."
+	icon_state = "cargoflag"
+
+/obj/item/flag/nation/med
+	name = "Medistan flag"
+	desc = "The flag of the independent, sovereign nation of Medistan."
+	icon_state = "medflag"
+
+/obj/item/flag/nation/sec
+	name = "Brigston flag"
+	desc = "The flag of the independent, sovereign nation of Brigston."
+	icon_state = "secflag"
+
+/obj/item/flag/nation/rnd
+	name = "Scientopia flag"
+	desc = "The flag of the independent, sovereign nation of Scientopia."
+	icon_state = "rndflag"
+
+/obj/item/flag/nation/atmos
+	name = "Atmosia flag"
+	desc = "The flag of the independent, sovereign nation of Atmosia."
+	icon_state = "atmosflag"
+
+/obj/item/flag/nation/command
+	name = "People's Republic of Commandzakstan flag"
+	desc = "The flag of the independent, sovereign nation of the People's Republic of Commandzakstan."
+	icon_state = "ntflag"
```

Comment:
> No flags/CTF elements

Aside from that, if you are going to have flags but have them not actually mean anything, I believe we still have departmental flags.